tests/contrib/django_util/test_views.py

```diff
@@ -180,9 +181,10 @@ def test_callback_works(self, pickle):
         self.assertEqual(
             response.status_code, django.http.HttpResponseRedirect.status_code)
         self.assertEqual(response['Location'], self.RETURN_URL)
+        jsonpickle_mock.decode.assert_called_once_with(pickled_flow)
```

Comment:
Done, yes now that you mention it setting a value to the return of a mock you just created *is* kind of silly. I'm guessing there was an iteration of the test that at some point made sense and then not-thoughtful refactors made it otherwise, thanks.